src/styles/components/widgets/_numeric-input.scss

```diff
@@ -49,6 +49,26 @@
   font-family: inherit;
 }
 
+.numeric-input__number--error {
+  display: inline-block;
+  border: var(--border-default);
+  border-color: var(--color-sienna);
+  background: var(--color-background-inputs);
+  cursor: text;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  text-align: left;
+  border-radius: var(--border-radius-small);
+  padding: var(--spacing-quarter-unit) var(--spacing-quarter-unit) var(--spacing-quarter-unit)
+    var(--spacing-half-unit);
+  width: 62px;
+  vertical-align: middle;
+  font-size: inherit;
+  color: inherit;
+  font-family: inherit;
+}
```

Comment:
We don't need to copy the whole class over, do we? We can just add a class with the difference.

Reply:
yeah, I could do `@extend .numeric-input__number;`